services/_base.md

```diff
@@ -22,3 +22,7 @@ add this report in frame along with the announce packet.
     }
 
 Notifies that the status code of the service changed.
+
+    const instance_name?: string @ instance_name
+
+A friendly name that describes the role of this service instance in the device.
```

Comment:
```suggestion
A friendly name that describes the role of this service instance in the device.
For example, `A` for button A, or `0` for servo channel 0.
Words like `left` should be avoided because of localization issues (unless they are printed on the device).
```